src/mod/l4_thruster.c

```diff
@@ -391,10 +537,41 @@ void ParseReadRequest(uint8_t* received_buffer,int len){
 
 
 
+void GeneralSetRequest_sequence(int argc, char *argv[]){
+	LAST_STARTED_MODULE = 1104;
+	/*
+	 * _sequence is a wrapper arround General  thruster registor Read/Set request functions
+	 *
+	 * it passes input argv further into original function
+	 *
+	 * However after execution of original function pointer of execution sequence stack THR_EXECUTION_INDEX
```

Comment:
Not  THR_EXECUTION_INDEX

but 
THR_HARDCODED_SEQUENCES[procedure_id].execution_index